dashboard/test/ui/test_status.css

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: Futura, "Trebuchet MS", Arial, sans-serif;
+}
+
+h2 {
+  font-family: monospace;
+  font-size: 1.1em;
+}
+
+h3 {
+  margin-bottom: 0.1em;
+}
+
+table {
+  width: 100%;
+}
+
+td {
+  padding-left: 0.3em;
+  min-width: 60px;
+}
+
+td.log-link {
+  text-align: center;
+}
+
+.centered-column {
+  width: 100%;
+  max-width: 1000px;
+  margin: auto;
+}
```

Comment:
This rule applies to the top-level wrapper div, centering all the content and limiting its width to 1000px (it tries to fit if the window is narrower than 1000px).